examples/mirror-demo/index.html

```diff
@@ -47,13 +48,77 @@
     <block type="variables_get"><field name="VAR">i</field></block>
     <block type="variables_get"><field name="VAR">j</field></block>
     <block type="variables_get"><field name="VAR">k</field></block>
-  </xml>
-
+  </xml> -->
```

Comment:
You're good to go ahead and remove this code instead of commenting it out! It isn't needed anymore :)